core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.spec.ts

```diff
@@ -239,7 +239,7 @@ describe('DotEditLayoutComponent', () => {
                 expect(component.pageState).toEqual(new DotPageRender(mockDotRenderedPage()));
             });
 
-            it('should save the layout after 10000', fakeAsync(() => {
+            it('should save the layout after DEBOUNCE_TIME', fakeAsync(() => {
```

Comment:
is it possible to use backticks for this?
something like:
`should save the layout after `${DEBOUNCE_TIME}`